Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM docker:17.12.0-ce-dind
+FROM docker:20.10.6-dind
```

Comment:
Nice 👍   Does this also work when the host OS / VM has support for cgroups v1 only, and is running an older version of Docker like Docker v18? I understand that Ubuntu v18.04 supports both cgroups v1 and v2, as the following command works:
```
sudo mount -t cgroup2 none /mnt/test
```
So it would probably have to be an older version of Ubuntu or some other distro, where cgroups v2 is not available.

I ask because, ideally, `balena preload` would work on both older and newer distros.

Reply:
You're right, I assumed that docker v20 has support for both cgroups v1 and v2 but I failed to test this. I'll see if I can find a distro that only supports v1 and if Docker 20 can run `hello-world` then I think we have our answer.

Reply:
Actually cgroups v2 has been in the kernel since 2014, so we really only need to test distros where v1 is the default and where v2 is the default.

> Fedora 31 was released on October 29, 2019. This is the first major distro that comes with cgroup v2 (aka unified hierarchy) enabled by default, 5 years after it first appeared in Linux kernel 3.16 (Aug 3, 2014).

Reply:
I confirmed this works on a `ubuntu/bionic64` VM image as well which is using cgroups v1 by default.